Sandbox/src/SandboxApp.cpp

```diff
@@ -1,9 +1,10 @@
 #include <Hazel.h>
+#include <Hazel/Core/Core.h>
```

Comment:
`Core.h` is now part of `Hazel.h` so we don't need to include it again.

Reply:
Cherno actually specifically said he wanted these logs gone, they aren't randomly being removed.

Reply:
We should probably look for other spots in the codebase where this was done.

Reply:
Yes, sandbox should indeed only include `Hazel.h` and the `entrypoint.h`. The only exception is for example to include some OpenGL file when we don't have it interfaced in Hazel yet.

Reply:
Resolved in https://github.com/TheCherno/Hazel/pull/165/commits/f1676f4378d5c57018bc5c598a8e7437200e6ba4, also removed the OpenGL specific includes